web-components/src/components/floating-modal/FloatingModal.ts

```diff
@@ -50,13 +51,23 @@ export namespace FloatingModal {
         if (this.container && this.show) {
           this.setContainerRect();
           this.setInteractInstance();
+          this.activateFocusTrap!();
+          this.setFocusableElements!();
+          this.focusInsideModal();
```

Comment:
set time out for this.focusInsideModal() like this:
`setTimeout(() => {
            this.focusInsideModal();
          }, 100);`